src/actions/breakpoints/index.js

```diff
@@ -79,6 +79,9 @@ export function enableBreakpointsInSource(cx: Context, source: Source) {
     for (const breakpoint of breakpoints) {
       if (breakpoint.disabled) {
         dispatch(enableBreakpoint(cx, breakpoint));
+      if (!getState.disabled) {
```

Comment:
I think we want to use a selector `getSkipPausing(getState())`

Reply:
used the selectors, let me know if this works :) @jasonLaster @darkwing @digitarald